SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3029,12 +3076,16 @@ USpatialActorChannel* USpatialNetDriver::CreateSpatialActorChannel(AActor* Actor
 {
 	// This should only be called from GetOrCreateSpatialActorChannel, otherwise we could end up clobbering an existing channel.
 
-	check(Actor != nullptr);
-	check(PackageMap != nullptr);
+	if (!ensureAlwaysMsgf(Actor != nullptr, TEXT("Tried to call CreateSpatialActorChannel for a nullptr Actor")))
+	{
+		return nullptr;
+	}
 
-	Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(Actor);
+	check(PackageMap != nullptr);
```

Comment:
Any reason this check is left in?

Reply:
PackageMap is a NetDriver member set at startup, so not a local issue.